.gitattributes

```diff
@@ -0,0 +1,21 @@
+# This file is part of Astarte.
```

Comment:
I think that these changes can be moved to a separate PR.

Reply:
This will fix an issue with the checks at compile time on Windows platforms.

Reply:
The binary cache was added because testing the PR took 7 minutes plus each time